test/e2e/api/atlas/atlas.go

```diff
@@ -71,5 +75,22 @@ func (a *Atlas) GetPrivateEndpoint(projectID, provider string) ([]mongodbatlas.P
 	if err != nil {
 		return nil, err
 	}
+	ginkgoPrettyPrintf(enpointsList, "listing private endpoints in project %s", projectID)
 	return enpointsList, nil
 }
+
+func (a *Atlas) GetAdvancedCluster(projectId, clusterName string) (*mongodbatlas.AdvancedCluster, error) {
+	advancedCluster, _, err := a.Client.AdvancedClusters.Get(context.Background(), projectId, clusterName)
+	if err != nil {
+		return nil, err
+	}
+	ginkgoPrettyPrintf(advancedCluster, "getting advanced cluster %s in project %s", clusterName, projectId)
```

Comment:
this function simulates what we are currently doing when displaying the output of mongocli when we make requests.